core/types/types.cc

```diff
@@ -172,8 +172,11 @@ TypePtr Types::dropSubtypesOf(Context ctx, const TypePtr &from, SymbolRef klass)
             }
         },
         [&](ClassType *c) {
+            auto cdata = c->symbol.data(ctx);
```

Comment:
can we move this after first `if`?
First if is the common case and it would be nice to not do unnecessary computations on path to it